jenkins-x-integration.yml

```diff
@@ -53,7 +53,7 @@ pipelineConfig:
                 command: make
                 args: ['build']
 
-              - name: test-integration
```

Comment:
I think makes sense to rename the `test-slow-integration` to just `test-integration`